src/components/tasks/card/index.tsx

```diff
@@ -369,6 +387,55 @@ const Card: FC<Props> = ({
         );
     };
 
+    const handleAssignment = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setAssigneeName(e.target.value);
+        if (e.target.value) setShowSuggestion(true);
+        else setShowSuggestion(false);
```

Comment:
nit: can we use the ternary operator `()? doSomething() : doSomething();` or curly braces `{}`